static/components/PlaceDetail.js

```diff
@@ -39,7 +39,7 @@ const PlaceDetail = (props) => {
       const metricValue = selectedPlace[metricName]
       const metricLabel = detailedInfectionMetricsMap[metricName]
       return(
-        <div style={{marginTop : 5, marginBottom: 5}}><b style={{alignSelf : "flex-start"}}>{metricLabel}</b> <span style={{color: "gold",alignSelf : "flex-end"}}>{getHaiStars(metricValue,3)}</span> </div>
+        <div style={{marginTop : 5, marginBottom: 5}}><b style={{alignSelf : "flex-start"}}>{metricLabel}</b> <span style={{color: "gold",alignSelf : "flex-end"}}>{getHaiEmoji(metricValue,3)}</span> </div>
```

Comment:
getHaiEmoji(metricValue,3) should be getHaiEmoji(metricValue). This is looks like a copy and paste artifact I missed. 

Or you could update getHaiEmoi to take an average score or scale since currently the individual metrics are on a scale of 1-3 (mean 2) and overall infection is relative and on a scale of 1-5 (mean 3). 

`const getHaiEmoji = (starCount, averageScore=3) => {
    if(!starCount){
        return ""
    }
    switch (true){
        case starCount < averageScore:
            return (<span style={{color:"green"}}>😄 Better Than Average</span>)
        case starCount == averageScore:
            return (<span style={{color:"orange"}}>😐 Average</span>)
        case starCount > averageScore:
            return (<span style={{color:"red"}}>🤢 Worse Than Average</span>)
    }
}`

So it would go to getHaiEmoji(metricValue,2) for the details and getHaiEmoji(metricValue,3) for the overall currently.

Reply:
Think that change is best handled after we make the decision on what to really show in all those cases. Also no need to add all that code in the comment. Description of your idea is sufficient.